etc/ReplayOfflineConfiguration.py

```diff
@@ -182,8 +195,8 @@
            global_tag=promptrecoGlobalTag,
            global_tag_connect=globalTagConnect,
            #archival_node="T0_CH_CERN_MSS",
-           tape_node="T0_CH_CERN_Disk",
-           disk_node="T0_CH_CERN_Disk",
+           #tape_node="T1_US_FNAL_MSS",
+           disk_node="T2_US_Vanderbilt",
            #raw_to_disk=False,
```

Comment:
You can put:
```suggestion
           raw_to_disk=True,
```
In the default dataset configuration (and any other PD for that matter)